### PR TITLE
hotfix - upgrade RealmSwift version

### PR DESCRIPTION
**Pull request by at-quangphu**

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ use_frameworks!
 platform :ios, '8.0'
 
 def shared_pods
-    pod 'RealmSwift', '~> 2.2'
+    pod 'RealmSwift', '~> 3.0.2'
```
Comment (zendobk) [inline Podfile]: `3.0`

File: RealmMapper.podspec [hunk 2]
```diff
@@ -9,7 +9,7 @@ Pod::Spec.new do |s|
     s.requires_arc = true
     s.ios.deployment_target = '8.0'
     s.ios.frameworks = 'Foundation', 'UIKit'
-    s.dependency 'RealmSwift', '~> 2.2'
+    s.dependency 'RealmSwift', '~> 3.0.2'
```
Comment (zendobk) [inline RealmMapper.podspec]: `3.0`

**Comment by at-quangphu** [comment]
@at-daonguyen Please help me review it. Thank you so much.

**Comment by at-daonguyen** [comment]
#14

**Pull request merged**